client/my-sites/theme/main.jsx

```diff
@@ -346,20 +347,18 @@ const ThemeSheet = React.createClass( {
 	},
 
 	renderSupportTab() {
-		if ( this.props.isCurrentUserPaid ) {
-			return (
-				<div>
-					{ this.renderContactUsCard( true ) }
-					{ this.renderThemeForumCard() }
-					{ this.renderCssSupportCard() }
-				</div>
-			);
-		}
+		let buttonCount = 1;
```

Comment:
We should destructure the props into locals to give more concise code below, like this:

`const { isCurrentUserPaid, isJetpack, forumUrl, isPremium, isWpcomTheme } = this.props;`